webapp/src/ts/app.component.ts

```diff
@@ -129,13 +139,6 @@ export class AppComponent {
     });
   }
 
-  private loadTranslations() {
-    this.translationsLoaded = this.translationLoaderService.getLocale().then(locale => {
```

Comment:
There's an error message in the console because you removed this:
```
feedback.service.ts:70 TypeError: Cannot read property 'then' of undefined
    at AppComponent.initForms (app.component.ts:311)
    at AppComponent.ngOnInit (app.component.ts:297)
```

Please always check that the console is error free.

Reply:
Fixed.